AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1521,6 +1521,7 @@ dmitripavlutin.com#?#article > div:has(> div > form[name="mc-embedded-subscribe-
 dmitripavlutin.com#?#main ~ aside > div > div:has(> form[name="mc-embedded-subscribe-form"])
 mtb-news.de##.mtbn-newsletter-bar
 kp.ru##.page-footer_subscribe
+m.hindustanttimes.com##.storyDetail > #loggedout
```

Comment:
I can see it in desktop, so, better `hindustantimes.com##.storyArea > div.storyDetail + div.subscribe-update`